### ISSUE TITLE
Are hyperparameters for BERT-base the same as BERT-large?

### ISSUE THREAD

**Issue by Ajanulk** [nlpdata/mrc_bert_baseline#3]
I have tried "--max_seq_length 512 --train_batch_size 24 --learning_rate 2e-5 --num_train_epochs 8.0 --output_dir dream_finetuned --gradient_accumulation_steps 3" with BERT-base-uncased, but can not get the result as shown in DREAM leaderboard "63.2"

My results are as follow
Dev: 63.5 Test: 62.2

Appreciate for any suggestion

**Comment by accreator** [comment]
Please just run the code several times more with different random seeds. As mentioned in the README, we run the experiments **five times with different random seeds** and report the best development set performance and the corresponding test set performance.

**Comment by Ajanulk** [comment]
> Please just run the code several times more with different random seeds. As mentioned in the README, we run the experiments **five times with different random seeds** and report the best development set performance and the corresponding test set performance.

Thanks! I get a try.
